openedx_caliper_tracking/tests/tests.py

```diff
@@ -6,10 +6,12 @@
 import json
 import os
 
-from django.test import TestCase
+from django.conf import settings
+from django.test import TestCase, override_settings, modify_settings
```

Comment:
`modify_settings` is not used anywhere so it should be removed.

Reply:
@asamolion

Reply:
@HamzaIbnFarooq Already fixed this. Check below this comment.